cfgov/v1/atomic_elements/organisms.py

```diff
@@ -588,6 +588,89 @@ def get_limit(self, value):
         return self.limit
 
 
+class SimpleChart(blocks.StructBlock):
+    title = blocks.CharBlock(required=True)
+    subtitle = blocks.CharBlock(required=False)
+    figure = blocks.CharBlock(required=False)
+
+    chart_type = blocks.ChoiceBlock(
+        choices=[
+            ('bar', 'Bar'),
+            ('datetime', 'Datetime'),
+            ('line', 'Line'),
+        ],
+        default='datetime',
+        required=True
+    )
+
+    data_source = blocks.TextBlock(
+        required=True,
+        help_text='URL of the chart\'s data source or an array of JSON data'
+    )
+
+    description = blocks.CharBlock(
```

Comment:
Where is this used? When I inspect a page with a chart, I don't see this description in the source.

Reply:
This was a bug! Fixed in 30b1bd56645e9f5948c8f8d04839408b6a91299b